docs/change-log.md

```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update January 20th, 2023
+
+- **New:** Humanitec's [Artefacts Service](https://api-docs.humanitec.com/#tag/ArtefactVersion) in addition to supporting branch and tag refs now also supports PR refs. 
+
+- **Fixed:** Issue preventing users from discarding a draft of a cloned deployment.
+
+- **Fixed:** [Service account](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-service-accounts#service-account) name on [Workload Details Screen](https://docs.humanitec.com/reference/user-interface/workload-details-screen) is no longer reset to `default` value after submiting the same name. 
+
+- **Improved:** Humanitec application's color palette has been update to better match the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) criteria.
```

Comment:
The color palette of the Humanitec UI has been updated to better match the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/)